interface/main/messages/messages.php

```diff
@@ -549,7 +549,7 @@ class='form-horizontal'
                                             <?php } else { ?>
                                                 <!-- This is for displaying a new note. -->
                                                 <button type="button" class="btn btn-primary btn-send-msg" id="newnote" value="<?php echo xla('Send message'); ?>"><?php echo xlt('Send message'); ?></button>
-                                                <button type="button" class="btn btn-cancel text-danger oe-opt-btn-separate-left" id="cancel" value="<?php echo xla('Cancel'); ?>"><?php echo xlt('Cancel'); ?></button>
+                                                <button type="button" class="btn btn-cancel btn-danger oe-opt-btn-separate-left" id="cancel" value="<?php echo xla('Cancel'); ?>"><?php echo xlt('Cancel'); ?></button>
```

Comment:
Per styling guide(https://github.com/openemr/openemr/issues/3160), seems like this should be `btn-secondary` instead of `btn-danger` for a Cancel button:

![image](https://user-images.githubusercontent.com/278968/78336021-8900a180-7543-11ea-9cad-17369204ff99.png)

![image](https://user-images.githubusercontent.com/278968/78336220-e3016700-7543-11ea-9fbf-8017466fafbc.png)

Reply:
Oops! I'll change the class to btn-secondary.